src/_css/main.css

```diff
@@ -13,12 +13,22 @@
 */
 
 :root {
+  color-scheme: dark light;
+  display: none;
```

Comment:
If we move this declaration to `:root.js { … }`, we can get rid of the `<noscript><style>`.

Reply:
Addressed in https://github.com/v8/v8.dev/pull/254/commits/3b32524d665aa734e362db26b9040280bd3881d0.